index.html

```diff
@@ -9,37 +9,54 @@
   <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
-  <link rel="stylesheet" href="custom.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Oxygen:700|Raleway&subset=latin-ext');
+  </style>
+
+  <link rel="stylesheet" href="css/custom.css" />
+
 </head>
 <body>
-<div class="container">
-  <nav class="navbar navbar-fixed-top" id="header">
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
-        <span class="sr-only">Toggle Navigation</span>
-      </button>
-      <a class="navbar-brand secondaryColor" href="/">Farmer's Market International</a>
+  <div class="container">
+    <nav class="navbar navbar-fixed-top navbar-inverse static-fixed" id="header">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+          <span class="sr-only">Toggle Navigation</span>
+        </button>
+        <a class="navbar-brand secondaryColor" href="/">Farmer's Market International</a>
 
     </div>
 
-    <ul class="nav navbar-nav navbar-right">
-      <li><a href="#join">Sign Up!</a></li>
-    </ul>
-    <ul class="nav navbar-nav navbar-right">
-      <li><a href="#solution">The Solution</a></li>
-    </ul>
-    <ul class="nav navbar-nav navbar-right">
-      <li><a href="#mission">Mission Statement</a></li>
+    <ul class="nav nav-tabs navbar-right">
```

Comment:
I had to do something because Join! was being cut off in the larger views and all you would ever see is Jo, I looked at the bootstrap site, and this is one of the solutions that were presented.